bigbluebutton-tests/playwright/parameters/parameters.spec.js

```
@@ -7,247 +7,246 @@ const { CreateParameters } = require('./createParameters');
+      await createParam.initUserPage(true, context, { testInfo });
       await createParam.mediaOnly();
     });
   });
 
   test.describe.parallel('Enforce Layout', () => {
-    test('CUSTOM_LAYOUT', async ({ browser, context, page }) => {
+    test('CUSTOM_LAYOUT', async ({ browser, context, page }, testInfo) => {
       const customParam = new CustomParameters(browser, context);
-      await customParam.initModPage(page, true, { createParameter: c.presentationFocus, joinParameter: c.enforceCustomLayout });
-      await customParam.initUserPage(true, context);
+      await customParam.initModPage(page, true, { createParameter: c.presentationFocus, joinParameter: c.enforceCustomLayout, testInfo, testInfo });
```

Biome (2.1.2)

[error] This property is later overwritten by an object member with the same name.

Overwritten with this property.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property.


(lint/suspicious/noDuplicateObjectKeys)